Makefile

```diff
@@ -81,6 +81,7 @@ schemas: schemas/gschemas.compiled
 	touch $@
 
 schemas/gschemas.compiled: schemas/*.gschema.xml
+	sudo cp schemas/*.xml /usr/share/glib-2.0/schemas && sudo glib-compile-schemas /usr/share/glib-2.0/schemas
```

Comment:
This will cause debian packaging to fail. You're not permitted to move files into system paths.

Reply:
Sure - I will check why the Gio.Settings API does not work but the command line does recognize a user path.
https://github.com/pop-os/shell/pull/554/files/117f4bc3e1e226ba35aa353852e1a4b83b075837#diff-e4512274660514546724494dd7ae0a43R36

Reply:
Looks like an oversight, I have to use GSettingsSchemaSource to build the object itself so it has the standard GSettings.Backend.